morphir-ir/shared/src/main/scala/zio/morphir/syntax/TypeSyntax.scala

```diff
@@ -1,34 +1,57 @@
 package zio.morphir.syntax
 
 import zio.morphir.ir.TypeModule.TypeCase.*
-import zio.morphir.ir.TypeModule.{Field, Type, TypeCase}
+import zio.morphir.ir.TypeModule.{Field, Type}
 import zio.morphir.ir.{FQName, Name}
 import zio.{Chunk, ZEnvironment}
 
 trait TypeSyntax {
   def defineVariable(name: String): Type[Any] = Type(VariableCase(Name.fromString(name)), ZEnvironment.empty)
   def defineVariable(name: Name): Type[Any]   = Type(VariableCase(name), ZEnvironment.empty)
-  def defineField(name: Name, fieldType: Type[Any]): Field[Type[Any]] = Field(name, fieldType)
-  def defineRecord(fields: Chunk[Field[Type[Any]]]): Type[Any]        = Type(RecordCase(fields), ZEnvironment.empty)
+
+  def defineField(name: Name, fieldType: Type[Any]): Field[Type[Any]]   = Field(name, fieldType)
+  def defineField(name: String, fieldType: Type[Any]): Field[Type[Any]] = Field(Name.fromString(name), fieldType)
+
+  def defineRecord(fields: Chunk[Field[Type[Any]]]): Type[Any] =
+    Type(RecordCase(fields), ZEnvironment.empty)
   def defineRecord(fields: Field[Type[Any]]*): Type[Any] =
     Type(RecordCase(Chunk.fromIterable(fields)), ZEnvironment.empty)
-  def defineTuple(elementTypes: Chunk[Type[Any]]): Type[Any] = Type(TupleCase(elementTypes), ZEnvironment.empty)
-  def defineTuple(first: Type[Any], second: Type[Any], rest: Chunk[Type[Any]]*): Type[Any] = ???
+
+  def defineTuple(elementTypes: Chunk[Type[Any]]): Type[Any] =
+    Type(TupleCase(elementTypes), ZEnvironment.empty)
+  def defineTuple(first: Type[Any], second: Type[Any], rest: Type[Any]*): Type[Any] =
+    Type(TupleCase(Chunk(first, second) ++ Chunk.fromIterable(rest)), ZEnvironment.empty)
+
   def defineFunction(paramTypes: Chunk[Type[Any]], returnType: Type[Any]): Type[Any] =
     Type(FunctionCase(paramTypes, returnType), ZEnvironment.empty)
-  def defineFunction[Annotations](paramTypes: Type[Annotations]*): TypeSyntax.DefineFunction[Annotations] =
-    new TypeSyntax.DefineFunction(() => Chunk.fromIterable(paramTypes))
+  def defineFunction[Annotations](paramTypes: Type[Annotations]*): SyntaxHelper.DefineFunction[Annotations] =
+    new SyntaxHelper.DefineFunction(() => Chunk.fromIterable(paramTypes))
+
   def defineExtensibleRecord(name: Name, fields: Chunk[Field[Type[Any]]]): Type[Any] =
     Type(ExtensibleRecordCase(name, fields), ZEnvironment.empty)
+  def defineExtensibleRecord(name: Name, fields: Field[Type[Any]]*): Type[Any] =
```

Comment:
I think if you actually use these you'll find that the compiler is going to complain, in order to get around this you will need to do this like `DefineFunction`